daemon/src/main.rs

```diff
@@ -0,0 +1,14 @@
+mod cfd;
```

Comment:
Should this file just live in `bin/taker.rs` perhaps so we can have two different binaries?

Reply:
yes, @klochowicz is working on that. We can also decide to merge at a later point (together with the maker ramp-up), but it keeps growing so I felt why not make a cut here